CMakeLists.txt

```diff
@@ -459,6 +459,14 @@ if(PDC_ENABLE_FASTBIT)
     set(ENABLE_FASTBIT 1)
 endif()
 
+#-----------------------------------------------------------------------------
+# Metadata with RocksDB
+#-----------------------------------------------------------------------------
+option(PDC_ENABLE_ROCKSDB "Enable RocksDB." OFF)
+if(PDC_ENABLE_ROCKSDB)
```

Comment:
Could we add a check for a miminum or required version of RocksDB? I'm assuming PDC will not work with any random version of it, so maybe we could add that as a check before enabling it to make it more robust?

Reply:
We have other dependencies and we do not perform minimum version check for those. But we document the tested version of dependencies. So, putting a few lines of instruction in the installation documentation about the RocksDB version and the commands and options for installing RocksDB should be more effective and efficient than writing a trivial and complex CMake find_package script.

Reply:
Added min version requirement